.travis.yml

```diff
@@ -1,63 +0,0 @@
-dist: trusty
-sudo: off
-language: python
-services:
-  - docker
-git:
-  depth: false
-addons:
-  apt:
-    packages:
-      - libssl-dev
-      - libffi-dev
-      - python-dev
-install: pip install --upgrade pip tox six
-cache: pip
-jobs:
-  include:
-    - stage: precheck
-      env: PURPOSE='Reject PRs to master'
-      script: ./scripts/ci/reject_master_prs.sh
-    - stage: precheck
-      env: PURPOSE='Dependency Check'
-      script: ./scripts/ci/dependency_check.sh
```

Comment:
where're equivalent of those jobs:  dependency check, test_profile_intergration, automation docker in new azure devops pipeline?

Reply:
1. dependency check -> Jobs: VerifyLinuxRequirements/VerifyWindowsRequirements/VerifyDarwinRequirements
2. test_profile_integration -> Job: IntegrationTestAgainstProfiles
3. Automation docker is useless, the container registry can be found. Current active container registry is managed by @fengzhou-msft